azure-iot-device/azure/iot/device/iothub/pipeline/pipeline_stages_iothub_mqtt.py

```diff
@@ -48,8 +51,14 @@ def _run_op(self, op):
             else:
                 client_id = op.device_id
 
-            username = "{hostname}/{client_id}/?api-version=2018-06-30".format(
-                hostname=op.hostname, client_id=client_id
+            query_params = [
+                ("api-version", pkg_constant.IOTHUB_API_VERSION),
+                ("DeviceClientType", pkg_constant.USER_AGENT),
```

Comment:
>DeviceClientType [](start = 18, length = 16)

So this becomes a different variable name here as opposed to `User-Agent` ?

Reply:
Yes. For IoTHub, in MQTT, the User-Agent is added as 'DeviceClientType', and in DPS it is added as 'ClientVersion'

---
In reply to: [307827069](https://github.com/Azure/azure-iot-sdk-python-preview/pull/147#discussion_r307827069) [](ancestors = 307827069)